assets/js/components/higherorder/withData.js~Stashed changes

```diff
@@ -0,0 +1,304 @@
+/**
```

Comment:
Looks like this file was a stashed change that was committed by mistake. Can you remove it please? 🙂

Reply:
Yes, will do, eslint failed several time trying to commit the file capitalisation changes. I'll reset and try in stages.

Reply:
This is failing a lot of the GitHub actions which it's odd because locally these file changes pass the linter. Does the merge successfully update files to the correct capitalisation on your system?